os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -462,6 +472,17 @@ static void recovery_user_assert(void)
 			binary_manager_exclude_rtthreads(tcb);
 		}
 
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
+		if (g_lib_binp) {
+			uint32_t start = (uint32_t)g_lib_binp->alloc[ALLOC_TEXT];
+			uint32_t end = start + g_lib_binp->textsize;
+			if (assert_pc >= start && assert_pc <= end) {
+				/* We will set binid to zero to indicate a crash in common library */
+				binid = 0;
```

Comment:
I think it can be okay for using 0, but you'd better use defined value for readability.

Reply:
Ok. Fixed.